Backend/src/facades/userFacade.ts

```diff
@@ -5,16 +5,20 @@ const bcrypt = require('bcryptjs');
 import { ApiError } from '../customErrors/apiError';
 import UserDataAccessorObject from '../dataAccessorObjects/userDAO';
 
-const UDAO = new UserDataAccessorObject();
 
-export default class UserFacade {
+export default class facade {
+
+    private _UDAO: UserDataAccessorObject;
+    constructor(schema: string) {
+        this._UDAO = new UserDataAccessorObject(schema);
+    }
```

Comment:
The difference is that you can have fields and you can add a constructor that forces the developer to provide certain values when creating an instance of the class. It is the more object oriented way of doing things which I felt like was needed here partly because the `UserFacade` is used in more than one place (in the `ResolverMap` and now in the test file).
The `UserFacade` used to be a singleton with all static methods but I changed that because we need different instances for different use cases. You can't create instances of functions that are returned by some module.
Think about how you would program this in Java - you'd end up with the same solution.